wbooks/src/redux/auth/actions.ts

```diff
@@ -1,18 +1,35 @@
 import { login } from '@services/AuthService';
-import { createTypes, completeTypes } from 'redux-recompose';
+import { createTypes, completeTypes, withPostSuccess } from 'redux-recompose';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+import Reactotron from 'reactotron-react-native';
 
-const completedActions = completeTypes({ primaryActions: ['LOGIN'] });
+const completedActions = completeTypes({ primaryActions: ['LOGIN', 'SET_CURRENT_USER'] });
 
 export const actions = createTypes(completedActions, '@@AUTH');
 
-const target = 'user';
```

Comment:
since you changed user, it should probably crash in each screen you were calling the reducer for the user data

Reply:
correct them

Reply:
I corrected them in this commit, I made sure that app doesn't crash because of this change.